docs/source-app/workflows/build_command_line_interface/index.rst

```diff
@@ -0,0 +1,55 @@
+############################
+Command-line Interface (CLI)
```

Comment:
why does this page exist twice?

Reply:
Because the bottom of the page is different.